packages/core/src/domain/session/sessionStore.ts

```diff
@@ -114,8 +113,15 @@ export function startSessionStore<TrackingType extends string>(
       sessionState = {}
     }
     if (hasSessionInCache()) {
+      // A session can be outdated if a new session has been generated in another browser tab
+      // if the new session is tracked, only expire the current session to restart after a user activity
+      // if the new session is not tracked, renew the session so that logs sdk stop sending events
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏What about adding a test for that?

Reply:
There are already tests covering these behaviours but they are splitting between RUM and Logs (which is logical). I've removed this comment and only added one specific to my change.

Reply:
> if the new session is not tracked, renew the session so that logs sdk stop sending events

is there a test for that?

Reply:
This use case `if the new session is not tracked, renew the session so that logs sdk stop sending events` 😅

FMU, It is the intent for some of the sessionStore changes but it is hard to guess it from the sessionStore changes only.

Reply:
This use case is not supported anymore.